rmgpy/molecule/molecule.pxd

```diff
@@ -248,6 +248,8 @@ cdef class Molecule(Graph):
 
     cpdef identify_ring_membership(self)
 
+    cpdef int count_aromatic_rings(self)
```

Comment:
Could you change the commit message to lower_case_with_underscores to be consistent with the code?
(same comment for the `Minor code improvements to ThermoDatabase.prioritizeThermo` commit)

Reply:
Done!